numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -928,25 +928,24 @@ def add_newdoc(place, name, doc):
 
     The Heaviside step function is defined as::
 
-                             0   if x < 0
-        heaviside(x, h0) =  h0   if x == 0
-                             1   if x > 0
```

Comment:
I might be inclined to leave this section as is, as the intent is far clearer. The arguments are positional anyway, so it's not incorrect.,.